### PR TITLE
refactor: remove usage of 'client.ToolchainObject'

### PR DESCRIPTION
**Pull request by xcoulon**
see https://github.com/codeready-toolchain/toolchain-common/pull/211

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -34,4 +34,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20210921100839-b43561751e72
```
Comment (rajivnathan) [inline go.mod:37]: Reminder

**Review by rajivnathan — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (xcoulon) [reply go.mod]: removed in https://github.com/codeready-toolchain/registration-service/pull/216/commits/f7a0b785647295320fb1d46ae3ba35a29296f90d

**Pull request merged**